### PR TITLE
[Docs] Reorgnize docs

### PR DESCRIPTION
**Pull request by dreamerlin**

### CODE REVIEW

File: docs/config.md [hunk 1]
```diff
@@ -1,19 +1,17 @@
...
-- [Config System](#config-system)
-    - [Config File Naming Convention](#config-file-naming-convention)
-    - [Config File Structure](#config-file-structure)
-        - [Config System for Action localization](#config-system-for-action-localization)
-            - [An Example of BMN](#an-example-of-bmn)
-        - [Config System for Action Recognition](#config-system-for-action-recognition)
-            - [An Example of TSN](#an-example-of-tsn)
-    - [FAQ](#faq)
-        - [Use intermediate variables in configs](#use-intermediate-variables-in-configs)
+We use python files as our config system. You can find all the provided configs under `$MMAction2/configs`.
```
Comment (innerlee) [inline docs/config.md:3]: We use python files as ~~our~~ config ~~system~~ files.

File: docs/useful_tools.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
+Apart from training/testing scripts, We provide lots of useful tools under the `tools/` directory.
+
```
Comment (innerlee) [inline docs/useful_tools.md:2]: toc for this file?

Comment (dreamerlin) [reply docs/useful_tools.md:2]: It will be ugly in readthedocs in this page if toc is added to this file.

Comment (innerlee) [reply docs/useful_tools.md:2]: screenshot?

Comment (dreamerlin) [reply docs/useful_tools.md:2]: ugly_readthedocs

Comment (innerlee) [reply docs/useful_tools.md:2]: How does the `# Tutorial 1: Finetuning Models` page looks like? It does not have second level items also

Comment (dreamerlin) [reply docs/useful_tools.md:2]: beaultiful_tutorial

Comment (innerlee) [reply docs/useful_tools.md:2]: They should be similar, right?

**Review by innerlee — APPROVED** [review]

**Pull request merged**